types/v4/index.d.ts

```diff
@@ -0,0 +1,147 @@
+// Type definitions for webpack-assets-manifest 3.0
+// Project: https://github.com/webdeveric/webpack-assets-manifest
+// Definitions by: Franklin Tse <https://github.com/FranklinWhale>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.7
+
+import { Plugin } from "webpack";
+import { SyncHook, SyncWaterfallHook } from "tapable";
+
+declare class WebpackAssetsManifest extends Plugin {
+    constructor(options?: WebpackAssetsManifest.Options);
+
+    /** https://github.com/webdeveric/webpack-assets-manifest#hooks */
+    hooks: {
+        apply: SyncHook<WebpackAssetsManifest>;
+
+        /**
+         * The `SyncWaterfallHook` class supports 3 type parameters only but this hook actually has 4 parameters. The type of 4th parameter is `AnyObject`.
+         *
+         * Refer to https://github.com/webdeveric/webpack-assets-manifest#hooks for details
+         */
+        customize: SyncWaterfallHook<WebpackAssetsManifest.Entry, WebpackAssetsManifest.AnyObject, WebpackAssetsManifest>;
```

Comment:
Any idea what arguments should be passed here? Webpack types allow 1-2, 3 are here though